src/DynamoCore/Interfaces/IRenderPackageFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Autodesk.DesignScript.Interfaces;
+
+namespace Dynamo.Interfaces
+{
+    public interface IRenderPackageFactory
+    {
+        int MaxTessellationDivisions { get; set; }
+        IRenderPackage CreateRenderPackage();
```

Comment:
This needs more documentation. What is tessellation?  Also, not totally sure it belongs on the interface...  Keep in mind that for applications where there is no notion of tessellation (e.g. 3d studio max) where everyone simply works with meshes, this value is useless.